supervision/detection/tools/inference_slicer.py

```diff
@@ -4,9 +4,10 @@
 
 import numpy as np
 
+from supervision.config import ORIENTED_BOX_COORDINATES as o_b_c
```

Comment:
Please import `from supervision.config import ORIENTED_BOX_COORDINATES` and skip `as o_b_c`. The name is long but descriptive. It is also consistent with the rest of the codebase.